industrial_ci/src/tests/black_check.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+function run_black_check() {
+  DOCKER_IMAGE="$DOCKER_BASE_IMAGE" ici_require_run_in_docker # this script must be run in docker
+
+  ici_time_start install_black
+  ici_quiet ici_asroot apt-get install -qq -y python3-pip
+  ici_quiet ici_asroot pip3 -q install black
```

Comment:
Is there any issue with running the python3 version of black with python2 code?

Reply:
Should be fine: https://github.com/psf/black#installation

Reply:
I assume it just checks the python shebang.